Runtime/Game/LootLockerSDKManager.cs

```diff
@@ -5092,6 +5092,65 @@ public static void SubmitScore(string memberId, int score, string leaderboardKey
 
             LootLockerAPIManager.SubmitScore(request, leaderboardKey, onComplete);
         }
+        /// <summary>
+        /// List the leaderboard archive.
+        /// </summary>
+        /// <param name="leaderboard_id">ID of the Leaderboard</param>
+        /// <param name="onComplete">onComplete Action for handling the response of type LootLockerLeaderboardHistoryResponse</param>
+        public static void ListLeaderboardArchive(int leaderboard_id, Action<LootLockerLeaderboardHistoryResponse> onComplete)
```

Comment:
Same comment as the leaderboard details one. Why add the int leaderboard_id version?